File: El Mamoun/Lv 1/Problem 06.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
 Problem 6: Write a program to ask the user to enter: • First Name • Last Name Then Print Full Name on screen.
 */


namespace Problem_6
{
    internal class Program
    {
        struct stPeronsInfo
        {
            public string FirstName;
            public string LastName;
        }
        static stPeronsInfo ReadPeronsInfo()
        {
            stPeronsInfo PeronsInfo;

            Console.Write("Please Enter you FirstName?: ");
            PeronsInfo.FirstName = Console.ReadLine();

            Console.Write("Please Enter you LastName?: ");
            PeronsInfo.LastName = Console.ReadLine();

            return PeronsInfo;

        }
        static string GetFullName(stPeronsInfo peronsInfo)
        {
            return peronsInfo.FirstName + " " + peronsInfo.LastName;
        }
        static void PrintResult(stPeronsInfo peronsInfo)
        {
            Console.Write("\nYou are Full Name is: " + GetFullName(peronsInfo));
        }


        static void Main(string[] args)
        {

            PrintResult(ReadPeronsInfo());
            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 06.cpp
#include<iostream>
using namespace std;

/*
 Problem 6: Write a program to ask the user to enter: • First Name • Last Name Then Print Full Name on screen.
 */

struct stPersonInfo
{
	string FirstName;
	string LastName;

};

stPersonInfo ReadPersonInfo()
{
	stPersonInfo PersonInfo;

	cout << "Please Enter Your First Name? ";
	cin >> PersonInfo.FirstName;

	cout << "Please Enter Your First Name? ";
	cin >> PersonInfo.LastName;

	return PersonInfo;
}

string GetFullName(stPersonInfo PersonInfo)
{
	string FullName = "";

	FullName = PersonInfo.FirstName + " " + PersonInfo.LastName;

	return FullName;
}

void PrintFullName(string FullName)
{

	cout << "\nYour Full Name is: " << FullName;

}


int main()
{

	PrintFullName(GetFullName(ReadPersonInfo()));


	return 0;
}